src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -300,6 +291,60 @@ protected override void ProcessRecord()
             }
         }
 
+        // All module extensions except ".psd1" are valid RootModule extensions
+        private static IList<string> validRootModuleExtensions = ModuleIntrinsics.PSModuleExtensions
```

Comment:
```suggestion
        private static readonly IList<string> s_validRootModuleExtensions = ModuleIntrinsics.PSModuleExtensions
```

Reply:
That's our misconfiguration of CodeFactor. We've discussed it before and agreed that `s_` is what we want. The `s_` convention is documented [here](https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md#naming-conventions) and there's [a PR open](https://github.com/PowerShell/PowerShell/pull/8500) to update CodeFactor.

Reply:
ok sounds good. I have just changed it to s_.